python/lsst/ts/hvac/mqtt_client.py

```diff
@@ -24,290 +24,85 @@
-        ----------
-        comando_encendido: `bool`
-            Switched on or off.
+    async def connect(self):
+        """Connect the client to the MQTT server.
         """
-        raise salobj.ExpectedError("Not implemented")
-
-    async def vec01P01(self, comando_encendido):
-        """Command VEC 01 on the first floor.
+        self.client = mqtt.Client()
+        self.client.on_message = self.on_message
+        self.client.connect(self.host, self.port)
+        self.client.loop_start()
+        self.client.subscribe(LSST_GENERAL_TOPIC)
+        self.log.info("Connected")
 
-        Parameters
-        ----------
-        comando_encendido: `bool`
-            Switched on or off.
+    async def disconnect(self):
+        """Disconnect the client from the MQTT server.
         """
-        raise salobj.ExpectedError("Not implemented")
+        self.client.loop_stop()
+        self.client.disconnect()
+        self.log.info("Disconnected")
 
-    async def vex03P04(self, comando_encendido):
-        """Command VEX 03 on the fourth floor.
+    def on_message(self, client, userdata, msg):
+        """Callback for when an MQTT message arrives.
 
         Parameters
         ----------
-        comando_encendido: `bool`
-            Switched on or off.
+        client: `mqtt.Client`
+            The client instance for this callback.
+        userdata:
+            The private user data as set in Client() or userdata_set(). May be
+            any data type according to the mqtt.Client API doc and is not set
+            nor used in this class.
+        msg: `mqtt.MQTTMessage`
+            The MQTT message that holds the topic and payload.
         """
-        raise salobj.ExpectedError("Not implemented")
+        self.msgs.put_nowait(msg)
 
-    async def vex04P04(self, comando_encendido):
-        """Command VEX 04 on the fourth floor.
+    def publish_mqtt_message(self, topic, payload):
+        """Publishes the specified payload to the specified topic on the MQTT
+        server.
 
         Parameters
         ----------
-        comando_encendido: `bool`
-            Switched on or off.
-        """
-        raise salobj.ExpectedError("Not implemented")
-
-    async def vin01P01(self, comando_encendido):
-        """Command VIN 01 on the first floor.
+        topic: `str`
+            The topic to publish to.
+        payload: `str`
+            The payload to publish.
 
-        Parameters
-        ----------
-        comando_encendido: `bool`
-            Switched on or off.
+        Returns
+        -------
+        is_published: `bool`
+            For now False gets returned since this functionality is disabled.
         """
-        raise salobj.ExpectedError("Not implemented")
+        self.log.error(
+            "This functionality is not enabled yet since this would start or stop "
+            "running systems or modify their configuration which is generally not a "
+            "good idea in operational systems."
```

Comment:
Absolutely. However, it is unclear at this moment if this CSC will be responsible for it or if this will be done via another way (for instance the user interface that comes with the MQTT/HVAC system). I have rewritten the statement to reflect this.